compiler/resolution/functionResolution.cpp

```diff
@@ -7936,6 +7928,31 @@ static void handleRuntimeTypes()
   insertRuntimeInitTemps();
 }
 
+//
+// If a symbol's type is not live, we can't use it.
+// Remove it if VarSymbol; remove containing fn if ArgSymbol.
+// Hopefully they are not used.
+//
+static void removeSymbolsWithRemovedTypes() {
+  std::vector<Symbol*> removedSyms; // for verification
+
+  for_alive_in_Vec(ArgSymbol, arg, gArgSymbols)
+    if (! arg->type->inTree())
+      if (FnSymbol* fn = toFnSymbol(arg->defPoint->parentSymbol)) {
+        removedSyms.push_back(fn);
+        fn->defPoint->remove();
+      }
+
+  for_alive_in_Vec(VarSymbol, var, gVarSymbols)
+    if (! var->type->inTree()) {
+      removedSyms.push_back(var);
+      var->defPoint->remove();
+    }
+
+  for_vector(Symbol, sym, removedSyms)
```

Comment:
Let's use { } with the loops in this function. Even though they are 1 statement, they are driven by macros.